python/tests/test_variable.py

```diff
@@ -425,3 +425,9 @@ def test_sum():
                            values=np.array([0.4, 0.8]),
                            unit=sp.units.m)
     assert sp.sum(var, Dim.Y) == expected
+
+
+def test_variance_acess():
+    v = sp.Variable()
+    assert v.variance is None
```

Comment:
Test `values` as well?

Reply:
Are the values always accessible? How you would get None for `values`?

Reply:
You constrct a variable using the default constructor. It is empty (I think an empty variable is not useful in practice, but possible).

Reply:
The default constructor in python would produce a variable with one 0 value...